packages/amplify-e2e-tests/src/__tests__/api_2.test.ts

```diff
@@ -21,7 +19,7 @@ import {
   getProjectMeta,
   getTransformConfig,
 } from 'amplify-e2e-core';
-import { TRANSFORM_CURRENT_VERSION, TRANSFORM_BASE_VERSION, writeTransformerConfiguration } from 'graphql-transformer-core';
```

Comment:
Is this change a part of this PR?

Reply:
Yes they were unused imports and tsc complains about them if you build the tests manually with yarn tsc and the e2e test package had build failures, I fixed those to make sure code is correct.